app/lib/world/GoalManager.coffee

```diff
@@ -96,6 +105,40 @@ module.exports = class GoalManager extends CocoClass
       @goalStates[goalID] = goalState
     @notifyGoalChanges()
 
+  # Adds any goals for the current capstoneStage
+  # Returns the current capstoneStage
+  addAdditionalGoals: (session, additionalGoals) ->
+    state = session.get('state') ? {}
+    if not state.capstoneStage
+      # In daily speak, we think of initial goals as stage 1 and additional goals
+      # as stage 2 and above. That is why we are starting from 2.
```

Comment:
Will `state.capstoneStage` be equal to 1 or undefined when the user is playing for initial goals?

Reply:
Yes, I agree - will look at adding it to make it easier to grok the code when reading 👍